hackathon_site/event/static/event/js/script.js

```diff
@@ -0,0 +1,63 @@
+// Change navbar color on scroll
```

Comment:
As mentioned above, what stuff in this file is needed for all pages and what is just for the landing page? Also, script.js is a bit generic